#!/usr/bin/env python
PACKAGE = "potbot_lib"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("potential_field_rows",                 int_t, 0, "A int parameter", 120, 1, 500)
gen.add("potential_field_cols",                 int_t, 0, "A int parameter", 120, 1, 500)
gen.add("potential_field_resolution",           double_t, 0, "A double parameter", 0.05, 0.01, 1)
gen.add("weight_attraction_field",              double_t, 0, "A double parameter", 0.1, 0, 10)
gen.add("weight_repulsion_field",               double_t, 0, "A double parameter", 0.1, 0, 10)
gen.add("distance_threshold_repulsion_field",   double_t, 0, "A double parameter", 0.5, 0, 2)

exit(gen.generate(PACKAGE, "potbot_lib", "PotentialField"))